asreviewcontrib/insights/algorithms.py

```diff
@@ -19,6 +20,14 @@ def _recall_values(labels, x_absolute=False, y_absolute=False):
     return x.tolist(), y.tolist()
 
 
+def _loss_value(labels):
+    positive_doc_ratio = sum(labels) / len(labels)
+    triangle_before_perfect_recall = positive_doc_ratio * 0.5
+    aera_under_recall_curve = metrics.auc(*_recall_values(labels))
```

Comment:
Isn't this not something like y*np.diff(x)?

In that case, we don't need the dependency (which is installed via asreview, but yeah).